functions/APIs/produce.js

```diff
@@ -0,0 +1,609 @@
+import React, { Component } from 'react'
```

Comment:
Please format with prettier (2 space indent)

Reply:
(Noted, fixed this on the other PR)